features/hack/on_feature_branch/in_uncommitted_subfolder.feature

```diff
@@ -19,17 +19,14 @@ Feature: git town-hack: starting a new feature from a new subfolder on the main
     Then it runs the commands
       | BRANCH      | COMMAND                      |
       | feature     | git fetch --prune --tags     |
-      | <none>      | cd {{ root folder }}         |
-      | feature     | git add -A                   |
+      |             | git add -A                   |
       |             | git stash                    |
       |             | git checkout main            |
       | main        | git rebase origin/main       |
       |             | git branch new-feature main  |
       |             | git checkout new-feature     |
       | new-feature | git stash pop                |
-      | <none>      | cd {{ folder "new_folder" }} |
     And I end up on the "new-feature" branch
-    And I am in the project root folder
```

Comment:
Shouldn't the tests still verify/specify in which folder we end up after running Git Town? I think that's important user-visible behavior.

Reply:
I'm uncertain if that is worth doing. We could certainly add it but then I think we need to change a bit how we execute the changes.

Currently we have the step: [^I run "([^"]+)" in the "([^"]+)" folder$](https://github.com/git-town/git-town/blob/master/test/steps.go#L238) which runs a command in a dedicated folder vs changing a working dir and then executing it. If we want to switch to that, I think we just need to break that step up.